collect_app/src/main/java/org/odk/collect/android/preferences/ServerPreferencesFragment.java

```diff
@@ -76,23 +78,68 @@
     //protected ExtendedEditTextPreference smsGatewayPreference;
     protected EditTextPreference submissionUrlPreference;
     protected EditTextPreference formListUrlPreference;
-    private ListPopupWindow listPopupWindow;
-    private Preference selectedGoogleAccountPreference;
-    private boolean allowClickSelectedGoogleAccountPreference = true;
 
     @Inject
     OpenRosaAPIClient openRosaAPIClient;
-
     @Inject
     GoogleAccountsManager accountsManager;
-
     @Inject
     Analytics analytics;
 
-    /*
-    private ListPreference transportPreference;
-    private ExtendedPreferenceCategory smsPreferenceCategory;
-    */
+    private ListPopupWindow listPopupWindow;
+    private Preference selectedGoogleAccountPreference;
+    private boolean allowClickSelectedGoogleAccountPreference = true;
+
+    public static ServerPreferencesFragment newInstance(boolean adminMode) {
```

Comment:
Might be good to have coverage for this admin mode behavior or at least flag it for QA.

Reply:
Great point. @getodk/testers for this PR it would be good to double check the server settings screen in admin mode!